firefox-ios/Shared/Strings.swift

```diff
@@ -182,6 +182,16 @@ extension String {
                 tableName: "Bookmarks",
                 value: "Delete Bookmark",
                 comment: "The title for the Delete Bookmark button, in the Edit Bookmark popup screen which is summoned from the main menu's Save submenu, which will delete the currently bookmarked site from the user's bookmarks.")
+            public static let SavedBookmarkToastLabel = MZLocalizedString(
+                key: "Bookmarks.Menu.SavedBookmarkToastLabel.v135",
+                tableName: "Bookmarks",
+                value: "Saved in \"%@\"",
+                comment: "The label displayed in the toast when saving a bookmark via the menu. The placeholder represents the folder where the bookmark is saved to")
+            public static let SavedBookmarkToastDefaultFolderLabel = MZLocalizedString(
+                key: "Bookmarks.Menu.SavedBookmarkToastDefaultFolderLabel.v135",
```

Comment:
I think it's 136 here too?
```suggestion
                key: "Bookmarks.Menu.SavedBookmarkToastLabel.v136",
                tableName: "Bookmarks",
                value: "Saved in \"%@\"",
                comment: "The label displayed in the toast when saving a bookmark via the menu. The placeholder represents the folder where the bookmark is saved to")
            public static let SavedBookmarkToastDefaultFolderLabel = MZLocalizedString(
                key: "Bookmarks.Menu.SavedBookmarkToastDefaultFolderLabel.v136",
```